package.json

```diff
@@ -29,9 +30,12 @@
     "node": ">= 0.10"
   },
   "dependencies": {
+    "@types/bluebird": "~3.0.36",
+    "@types/express": "~4.0.34",
```

Comment:
I checked many repositories and everywhere @types dependencies places in devDependencies. I think if I use only es5 or es6 why I need install types for typescript?